easybuild/easyconfigs/t/Trim_Galore/Trim_Galore-0.6.7-GCCcore-10.3.0.eb

```
@@ -0,0 +1,43 @@
+# Uploaded by J. Sassmannshausen NHS/GSTT
+
+easyblock = 'Tarball'
+
+name = 'Trim_Galore'
+version = '0.6.7'
+
+homepage = 'https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/'
+description = """Trim Galore! is a wrapper script to automate quality and adapter
+trimming as well as quality control, with some added functionality to remove biased
+methylation positions for RRBS sequence files (for directional, non-directional 
+(or paired-end) sequencing)."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['https://github.com/FelixKrueger/TrimGalore/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = [
+    '49b4622c9f839f8a7037fa2f7de9ab873e4fb702f29c14b4d7f095abf6738f74',  # 0.6.7.tar.gz
+]
+
+dependencies = [
+    ('Perl', '5.32.1'),
+    ('cutadapt', '3.4'),
+    ('FastQC', '0.11.9', '-Java-11', True),
+]
```

The script `trim_galore` uses python and pigz as well. We should keep the same list of dependencies as in version 0.6.6: 
```suggestion
dependencies = [
    ('Python', '3.9.6'),
    ('Java', '11', '', True),
    ('pigz', '2.6'),
    ('Perl', '5.32.1'),
    ('cutadapt', '3.4'),
    ('FastQC', '0.11.9', '-Java-%(javaver)s', True),
]
```

---

Thanks for this. I have added the additional packages as requested. For Python: I think the only version for the `GCCcore-10.3.0` toolchain is version `2.6.5` as far as I could see. Please correct me if I am wrong.